app/packs/controllers/review_workflow_controller.js

```diff
@@ -0,0 +1,17 @@
+import { Controller } from "stimulus"
+
+export default class extends Controller {
+  static targets = [ "enabled", "reviewers" ]
+
+  connect() {
+    this.toggle()
+  }
+
+  toggle() {
```

Comment:
Is there a better name?  Toggle to me means flip the state, but this is really just synchronizing with the UI.

Reply:
It is toggling disabled. But I'm open to other suggestions.